server/base/src/test/java/org/apache/accumulo/server/util/AdminTest.java

```diff
@@ -93,4 +115,94 @@ public void testCannotQualifySessionId() {
     EasyMock.verify(zc);
   }
 
+  @Test
+  public void testDanglingFate() {
```

Comment:
nice test. Includes test where all are inactive fateIds, a mix of active and inactive, and tests the race condition

Reply:
Updated the unit test in 987f699